jd-geek-source/src/app/components/test-diplay/test-diplay.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h4>Testing Getting Data From The Best Buy API</h4>
+
+  <div *ngFor="let product of testResults">
+    <div>
+      <p>
+        <b>Department: </b>{{ product.department }}&nbsp;&nbsp; <b>SKU: </b
+        >{{ product.sku }}
+      </p>
+      <p><b>Name: </b>{{ product.name }}</p>
+      <p><b>Sale Price: </b>{{ product.salePrice }}</p>
+    </div>
+  </div>
+
+  <div>
```

Comment:
Empty `div` doesn't appear to serve a purpose. It's important to keep markup as clean/minimal as possible.

Reply:
this component will not be in the next iteration.  It was merely to show some data being pulled from the BB api